huaweicloud/services/evs/resource_huaweicloud_evs_snapshot_group.go

```diff
@@ -0,0 +1,517 @@
+// @API EVS POST /v5/{project_id}/snapshot-groups/{snapshot_group_id}/tags/delete
+func ResourceEvsSnapshotGroup() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceEvsSnapshotGroupCreate,
+		ReadContext:   resourceEvsSnapshotGroupRead,
+		UpdateContext: resourceEvsSnapshotGroupUpdate,
+		DeleteContext: resourceEvsSnapshotGroupDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(10 * time.Minute),
+			Update: schema.DefaultTimeout(10 * time.Minute),
+			Delete: schema.DefaultTimeout(5 * time.Minute),
+		},
+
+		CustomizeDiff: config.FlexibleForceNew(snapshotGroupNonUpdatableParams),
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"server_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"volume_ids": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"instant_access": {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"tags": {
+				Type:     schema.TypeMap,
+				Optional: true,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"enterprise_project_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"incremental": {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
+			"created_at": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"status": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"updated_at": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"enable_force_new": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.StringInSlice([]string{"true", "false"}, false),
+				Description:  utils.SchemaDesc("", utils.SchemaDescInput{Internal: true}),
+			},
+		},
+	}
+}
+
+func buildCreateSnapshotGroupBodyParams(d *schema.ResourceData, epsID string) map[string]interface{} {
+	body := map[string]interface{}{
+		"server_id":             utils.ValueIgnoreEmpty(d.Get("server_id")),
+		"volume_ids":            utils.ValueIgnoreEmpty(d.Get("volume_ids")),
+		"instant_access":        utils.ValueIgnoreEmpty(d.Get("instant_access")),
+		"name":                  utils.ValueIgnoreEmpty(d.Get("name")),
+		"description":           utils.ValueIgnoreEmpty(d.Get("description")),
+		"enterprise_project_id": utils.ValueIgnoreEmpty(epsID),
+		"incremental":           utils.ValueIgnoreEmpty(d.Get("incremental")),
+	}
+	if tags, ok := d.Get("tags").(map[string]interface{}); ok && len(tags) > 0 {
```

Comment:
If the document states that it has a default value, it is recommended not to actively specify the default value.
```
if d.Get("instant_access").(bool) {
  body["instant_access"] = true.
}
```